contrib/checkbox-ce-oem/checkbox-ce-oem-snap/series_classic24/snap/snapcraft.yaml

```
@@ -0,0 +1,103 @@
+name: checkbox-ce-oem
+summary: Checkbox CE OEM test runner and public providers for 24.04 classic
+description: "Checkbox CE OEM test runner and public providers for 24.04 classic"
+confinement: classic
+grade: stable
+
+version: '1.0-jammy'
```

This should be Noble
```suggestion
version: '1.0-noble'
```